evap/staff/importers.py

```diff
@@ -764,21 +812,42 @@ def process_participants(self, evaluation, test_run, user_list):
 
         if already_related:
             msg = format_html(
-                _("The following {} users are already participants in evaluation {}:"),
-                len(already_related),
-                evaluation.full_name,
+                ngettext(
+                    "The following user is already participant in evaluation {name}:",
+                    "The following {user_count} are already participants in evaluation {name}:",
+                    len(already_related),
+                ).format(user_count=len(already_related), name=evaluation.full_name)
```

Comment:
... and here, it is a vulnerability, because `evaluation.full_name` is not escaped anymore and could contain malicious contents. Pls fix :)